apps/backend/src/libs/plugins/authorization/authorization.plugin.ts

```diff
@@ -28,15 +28,22 @@ const authorization = fp<Options>((fastify, options, done) => {
 			}
 		}
 
-		const jwtToken = request.headers["authorization"];
+		const authHeader = request.headers["authorization"];
 
-		if (!jwtToken) {
+		if (!authHeader) {
 			throw new AuthError({
 				message: ExceptionMessage.NO_TOKEN_PROVIDED,
 				status: HTTPCode.UNAUTHORIZED,
 			});
 		}
 
+		const BEARER_PREFIX = "Bearer ";
+		const BEARER_PREFIX_LENGTH = BEARER_PREFIX.length;
+
+		const jwtToken = authHeader.startsWith(BEARER_PREFIX)
```

Comment:
I may be wrong, but in our case authorization header always starts with "Bearer ", so there is no need in this check

Reply:
We don't know whether this is a request we send from our frontend, so we should check. Also, backend doesn't know anything about how it's implemented on the frontend.